serverlessworkflow/modules/ROOT/pages/tooling/serverless-workflow-editor/swf-editor-vscode-extension.adoc

```diff
@@ -0,0 +1,191 @@
+* *Run On Save*: Executes a command to generate an SVG of the workflow when a _save_ action is performed.
+* *Service Registries*: Provides a link to access the _settings.json_, to configure the Serivce Registries.
+* *Should Reference Service Registry Functions With Urls*: Provides a check box to toggle between downloading the openAPI file or use the url of the function in Service Registry.
+* *Specs Storage Path*: Provides a text box to edit the path of openAPI spec file(defaults to 'specs' folder).
+* *Svg Filename Template*: Provides a text box to edit the filename template to be used when generating SVG files.
+* *Svg File Path*: Provides a text box to edit the path to generate SVG files.
+
+.Settings page of the extension
+image::tooling/serverless-workflow-editor/swf-editor-vscode-settings.png[]
+
+=== Context based auto-completion
+The extension can use link:{open_api_spec_url}[openAPI spec] files stored locally in the 'specs folder'(<<Kogito Serverless Workflow Editor Setttings,check the settings section>>) or in remote Service Registries to provide advanced auto-complete features for *Function Definitions*.
+
+The extension is able to parse the openAPI spec files and provide the list of Function Definitions in the auto-complete dialog.
+
+.Function Definition auto complete dialogue
+image::tooling/serverless-workflow-editor/swf-editor-function-definition-auto-complete-dialogue.png[]
+
+After selecting one of the options in the auto-complete dialog, a full Function Definition object will be added to the functions array with the right settings.
+
+.Function Definition object auto completed
+image::tooling/serverless-workflow-editor/swf-editor-auto-completed-function-definition.png[]
+
+Once function definition is added, it will be available for auto completion in the Function Reference section.
+
+.Function Reference auto complete dialogue
+image::tooling/serverless-workflow-editor/swf-editor-auto-complete-function-reference-dialogue.png[]
+
+After selecting one of the options in the auto-complete dialog, a full Function Reference object will be added with the right reference name and attributes.
+
+.Function Reference object auto completed
+image::tooling/serverless-workflow-editor/swf-editor-auto-completed-function-reference.png[]
+
+[NOTE]
+====
+After selecting one of the options in the auto-complete dialog, the openAPI spec file will be downloaded into the 'specs' folder(<<Kogito Serverless Workflow Editor Setttings,Configured in the settings>>). The saved file follows the format of _REGISTRY-NAME_OPERATION-NAME_VERSION_.
+====
+
+If any change is made in the registry, make use of the *Refresh Service Registries*... to fetch the new changes.
+
+.Refresh Service Registries button in Function Definition section
+image::tooling/serverless-workflow-editor/swf-editor-refresh-registry.png[]
+
+Click on the *Add function*... button to create a new Function Definition object.
+
+=== Service Registry settings
```

Comment:
I think we may have to change a bit the contents of this section, we could probably consolidate it with the `Connecting the extension to Apicurio Service Registries`